Disable input prompts with fixed answers for installation of Net-SSLeay
author: Alex Domingo (Vrije Universiteit Brussel)
--- Makefile.PL.orig	2024-12-21 11:29:33.708553000 +0100
+++ Makefile.PL	2024-12-21 11:37:16.495964375 +0100
@@ -28,11 +28,7 @@
 # the Net-SSLeay loadable object on Windows
 my $win_link_statically = 0;
 
-my $tests = prompt(
-  "Do you want to run external tests?\n".
-  "These tests *will* *fail* if you do not have network connectivity.",
-  'n',
-) =~ /^y/i ? 't/*/*.t t/*/*/*.t' : 't/local/*.t t/handle/local/*.t';
+my $tests = 't/local/*.t t/handle/local/*.t';
 
 my %eumm_args = (
   NAME => 'Net::SSLeay',
